Cython/Compiler/FusedNode.py

```diff
@@ -579,6 +580,12 @@ def make_fused_cpdef(self, orig_py_func, env, is_def):
         decl_code = Code.PyxCodeWriter(context=context)
         decl_code.put_chunk(
             u"""
+                cdef union {{lecheck_cname}}:
+                    unsigned int i
```

Comment:
Fixed in https://github.com/cython/cython/pull/1607/commits/55bbef398595d760c1d2736809f4a10f334a98cc